packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -128,7 +128,7 @@ const removeOldAuthProvider = async (content) => {
 
 // check to make sure AuthProvider doesn't exist
 const checkAuthProviderExists = () => {
-  const content = fs.readFileSync(WEB_SRC_INDEX_PATH).toString()
+  const content = fs.readFileSync(WEB_SRC_APP_PATH).toString()
```

Comment:
```suggestion
  const content = fs.readFileSync(WEB_SRC_APP_PATH, 'utf8')
```

Reply:
If the encoding is specified then this will return a string rather than a buffer.

Reply:
Let me change these afterwards @peterp just to minimise changes if its ok?